tmpl/wiki/tutorial-lwt.md

```diff
@@ -2,7 +2,7 @@
 
 ##Basics
 
-The full Lwt manual is available [elsewhere](http://ocsigen.org/lwt/manual/), but the minimal stuff needed to get started is here.
+The full Lwt manual is available [elsewhere](https://ocsigen.org/lwt/5.1.2/manual/manual), but the minimal stuff needed to get started is here.
```

Comment:
Looks like https://ocsigen.org/lwt would work here (that's what the site's header bar uses).
We should probably use https everywhere else in this file too.

Reply:
https://ocsigen.org/lwt redirects to http://ocsigen.org/lwt...

But https://ocsigen.org/lwt/5.2.0/manual/manual does not :-)

They should probably revise their server configuration... (and enforce https)
Meanwhile is it better to have a generic link that may survive the future versions but redirects to a non-secrure version of their site ? or an `https` one tied to a specific version of `lwt` ?

Reply:
I switched to https links and will open an issue at ocsigen.